routing_model/index.html

```diff
@@ -102,6 +102,15 @@ <h2>Node's entire lifetime</h2>
 
 
   <h2>Node as a member of a section</h2>
+  <p>Once a node has joined a section, they need to be ready to take on multiple roles simultaneously:<br>
+  <ul>
+    <li> Deal with rellocation:<br>
+         The section as a whole will sometimes act as the source and sometimes as the destination of rellocations. This means nodes will sometimes rellocate away from this section and sometimes rellocate to this section. Nodes in this section will need to perform the right flow to deal with both of these situations.
+    <li> Decide who's an elder or a plain old adult
+  </ul>
+  All of these flows are happening simultaneously, but really, only one flow is active at anytime and all others must be in a "wait" state.<br>
```

Comment:
I am not sure it is clear that for example, a node can handle relocating a node to its section at the same time it is relocating a node away from its section.

Reply:
Uhm. I wasn't 100% clear on this, but I thought that's what you meant. So what about:
> All of these flows are happening simultaneously, but in most cases, only one flow is active at anytime while the others are in a "wait" state.

Or would you just remove the entire sentence?

Reply:
> All of these flows are happening simultaneously, but in most cases, only one flow is active at anytime while the others are in a "wait" state.

No, in all case, only one flow is active at a time, but there are multiple events to see and wait for in a flow, so you may have sent a RelocateResponse and are resource proofing, but in the wait, you may see the relocation trigger and send yourself an ExpectCandidate.

Reply:
I'm happy with the sentence as it is.